lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -235,6 +241,30 @@ private function getRelation($sourceActorId, $targetActorId, $relationTypeId)
     return QubitPdo::fetchColumn($sql, $paramsVariant);
   }
 
+  /**
+   * Use SQL to fetch relations
+   *
+   * @param int $sourceActorId  ID of an actor that possibly relates to others
+   * @param int $targetActorId  ID of an actor that possibly relates to others
+   *
+   * @return array  array of relation IDs
+   */
+  private function getRelations($sourceActorId, $targetActorId)
+  {
+    $sql = "SELECT id FROM relation
+      WHERE subject_id = :subject_id
+      AND object_id = :object_id";
+
+    $params = [
```

Comment:
Nitpick: You could avoid the $params variable assignment and define the parameter array in the `QubitPdo::fetchAll()` call.

Reply:
Nice... done!